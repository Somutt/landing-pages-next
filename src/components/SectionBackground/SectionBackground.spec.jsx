import { screen } from '@testing-library/react';
import { SectionBackground } from '.';

import { renderTheme } from '../../utils/renderTheme';

describe('<SectionBackground />', () => {
  it('should render SectionBackground with dark background', () => {
    const { container } = renderTheme(
      <SectionBackground background={true}>
        <h1>Teste</h1>
      </SectionBackground>,
    );
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render SectionBackground with light background', () => {
    const { container } = renderTheme(
      <SectionBackground>
        <h1>Teste</h1>
      </SectionBackground>,
    );
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
