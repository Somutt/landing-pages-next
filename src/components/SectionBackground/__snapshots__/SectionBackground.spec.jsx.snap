// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground /> should render SectionBackground with dark background 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 4.0rem;
  width: 100%;
}

<div
  class="c0"
  id="id-1995-0843329079482"
>
  <div
    class="c1"
  >
    <h1>
      Teste
    </h1>
  </div>
</div>
`;

exports[`<SectionBackground /> should render SectionBackground with light background 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: flex;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 4.0rem;
  width: 100%;
}

<div
  class="c0"
  id="id-9245-602476291017"
>
  <div
    class="c1"
  >
    <h1>
      Teste
    </h1>
  </div>
</div>
`;
